i18n/locales/zh-Hant.json

```
@@ -1,7 +1,10 @@
 {
+  "account_page_update_settings_error": "未能更新帳戶設定。",
   "app_header_shelf": "書架",
   "app_header_store": "書店",
   "bookshelf_more_menu_title": "更多選項",
+  "bookshelf_no_items": "你的書架為空",
```

```suggestion
  "bookshelf_no_items": "你的書架沒有書",
```
or just simply `你的書架是空的`